src/projectworks.ts

```diff
@@ -0,0 +1,45 @@
+import fetch, { Headers, Response } from 'node-fetch';
+import { URLSearchParams } from 'url';
+import type { ProjectWorksLeave, ProjectWorksUser } from './types';
+
+const PW_URL = 'https://api.projectworksapp.com';
+const PW_USERNAME = process.env.PROJECTWORKS_USERNAME;
+const PW_PASSWORD = process.env.PROJECTWORKS_PASSWORD;
+
+const apiGet = async (path: string): Promise<Response> => {
+  const headers = new Headers();
+
+  headers.append('Accept', 'application/json');
+  headers.append(
+    'Authorization',
+    `Basic ${Buffer.from(
+      `${PW_USERNAME as string}:${PW_PASSWORD as string}`
```

Comment:
🐘 you can tell TypeScript that these variables will always be set in the environment by putting this at the top level:

```ts
declare global {
  namespace NodeJS {
    interface ProcessEnv {
      PROJECTWORKS_USERNAME: string;
      PROJECTWORKS_PASSWORD: string;
    }
  }
}
```